JetMETCorrections/Configuration/python/JetCorrectorsForReco_cff.py

```diff
@@ -6,6 +6,5 @@
 from JetMETCorrections.Configuration.JetCorrectors_cff import ak4PFCHSL1FastL2L3CorrectorChain,ak4PFCHSL1FastL2L3CorrectorTask
```

Comment:
Sorry for the late response to this. 
I missed changing ak4PFCHSL1FastL2L3Corrector**Chain** to ak4PFCHSL1FastL2L3Corrector**Task**.

I will fix it.